components/feature/customtabs/src/main/java/mozilla/components/feature/customtabs/CustomTabsToolbarFeature.kt

```diff
@@ -138,7 +144,10 @@ class CustomTabsToolbarFeature(
         }.also { items ->
             val combinedItems = menuBuilder?.let { builder -> builder.items + items } ?: items
 
-            toolbar.setMenuBuilder(BrowserMenuBuilder(combinedItems))
+            val extras = mapOf(
+                    "customTab" to true
+            )
+            toolbar.setMenuBuilder(BrowserMenuBuilder(combinedItems, extras))
```

Comment:
We may end up overriding extras the app has provided.

* Do we need to do anything here? If we allow the app to specific extras then it can attach the flag for us already.
* If we always want to provide this flag ourselves from the feature then let's combine the extras with the already existing ones (like we do for menu items). :)

Reply:
We don't actually need any of the app's `extras` so I've just chosen to rename this variable so we're not overriding it.